src/interface/obsidian/src/chat_modal.ts

```diff
@@ -1,6 +1,6 @@
 import { App, Modal, request, Setting } from 'obsidian';
 import { KhojSetting } from 'src/settings';
-
+import fetch from "node-fetch";
```

Comment:
This should work for locally-hosted Khoj + Obsidian clients. @debanjum , I'll need to enlist your help for Emacs support 😁

Reply:
Let me try add streaming to `khoj.el` for Emacs separately?